charts/renku/requirements.yaml

```diff
@@ -1,29 +1,30 @@
 dependencies:
 - name: renku-ui
   alias: ui
-  version: "0.1.0-c776f4e"
+  version: "0.1.0-0bc7625"
```

Comment:
These are upgrades, not downgrades?

Reply:
yes -- were they supposed to be downgrades?

Reply:
nah, i just didn't check the git logs to verify that